symplyphysics/core/vectors/arithmetics.py

```diff
@@ -38,37 +38,52 @@ def equal_vectors(vector_left: Vector, vector_right: Vector) -> bool:
     return True
 
 
-# Sum of two vectors
-# Sum of two vectors can be seen as a diagonal of the parallelogram, where vectors are adjacent sides of this parallelogram.
-# To subtract vectors, multiply one of the vectors to -1 and add them.
+# Add any number of vectors
 #NOTE: adding two non-cartesian vectors is not a trivial task. We suggest to convert them to cartesian
 #      vectors, add them and convert back.
-def add_cartesian_vectors(vector_left: Vector, vector_right: Vector) -> Vector:
-    if vector_left.coordinate_system != vector_right.coordinate_system:
-        raise TypeError(
-            f"Different coordinate systems in vectors: {str(vector_left.coordinate_system)} vs {str(vector_right.coordinate_system)}"
-        )
-    if vector_left.coordinate_system.coord_system_type != CoordinateSystem.System.CARTESIAN:
-        coord_name_from = CoordinateSystem.system_to_transformation_name(
-            vector_left.coordinate_system.coord_system_type)
-        raise ValueError(
-            f"Addition is only supported for cartesian coordinates: got {coord_name_from}")
-    if vector_right.coordinate_system.coord_system_type != CoordinateSystem.System.CARTESIAN:
-        coord_name_from = CoordinateSystem.system_to_transformation_name(
-            vector_right.coordinate_system.coord_system_type)
-        raise ValueError(
-            f"Addition is only supported for cartesian coordinates: got {coord_name_from}")
-    (list_left_extended, list_right_extended) = _extend_two_vectors(vector_left, vector_right)
-    result = list(
-        map(lambda lr: sympify(lr[0] + lr[1]), zip(list_left_extended, list_right_extended)))
-    return Vector(result, vector_left.coordinate_system)
-
-
-def subtract_cartesian_vectors(vector_left: Vector, vector_right: Vector) -> Vector:
-    return add_cartesian_vectors(
-        vector_left,
-        scale_vector(-1, vector_right)
-    )
+def add_cartesian_vectors(*vectors: Vector) -> Vector:
```

Comment:
Use `Sequence[Vector]`

Reply:
That means we would need to change the code using this function throughout the whole codebase that uses two arguments instead of one sequence argument.

Reply:
Hm, you may be right.